ui/page/dcrdex/dcrdex_page.go

```diff
@@ -51,13 +52,14 @@ func (pg *DEXPage) ID() string {
 // Part of the load.Page interface.
 func (pg *DEXPage) OnNavigatedTo() {
 	pg.ctx, pg.ctxCancel = context.WithCancel(context.TODO())
-
-	if pg.CurrentPage() == nil {
-		// TODO: Handle pg.inited
+	// TODO: set pg.inited
+	if !pg.inited {
 		pg.Display(NewDEXOnboarding(pg.Load))
+	} else if pg.CurrentPage() == nil {
+		pg.Display(NewDEXMarketPage(pg.Load))
+	} else {
+		pg.CurrentPage().OnNavigatedTo()
 	}
```

Comment:
Check if `pg.CurrentPage() != nil` first, otherwise, everytime this page is navigated to, the current page will always be discarded and a new onboarding page will be displayed if not inited.